src/DependencyInjection/Configuration.php

```diff
@@ -32,7 +37,13 @@ public function getConfigTreeBuilder()
 
         $node
             ->children()
-                ->scalarNode('db_driver')->isRequired()->end()
+                ->scalarNode('db_driver')
+                    ->defaultValue('no_driver')
```

Comment:
Please add a call to `info()`

Reply:
Added `->info('Choose persistence mechanism driver from the following list: "doctrine_orm", "doctrine_mongodb", "doctrine_phpcr"')`